rs/nervous_system/canisters/src/registry.rs

```diff
@@ -42,23 +42,22 @@ impl Default for RegistryCanister {
 #[async_trait]
 impl Registry for RegistryCanister {
     async fn get_latest_version(&self) -> Result<RegistryVersion, NervousSystemError> {
-        let response: Result<Vec<u8>, (i32, String)> =
-            CdkRuntime::call_bytes_with_cleanup(self.canister_id, "get_latest_version", &[]).await;
-        response
-            .map_err(|(code, msg)| {
+        let reply: Vec<u8> = Call::unbounded_wait(self.canister_id.into(), "get_latest_version")
```

Comment:
Why did `CdkRuntime::call_bytes_with_cleanup` change to `Call::unbounded_wait`?

Reply:
This is now moot.

I was able to get rid of all these changes by making all our stuff use the same version of `ic-cdk` (namely, 0.17.1).